hydra-node/src/Hydra/Chain/Direct/Wallet.hs

```diff
@@ -239,8 +240,9 @@ coverFee_ pparams systemStart epochInfo lookupUTxO walletUTxO partialTx@Validate
   let inputs' = inputs body <> Set.singleton input
   resolvedInputs <- traverse resolveInput (toList inputs')
 
+  let utxo = lookupUTxO <> walletUTxO
   estimatedScriptCosts <-
-    estimateScriptsCost pparams systemStart epochInfo (lookupUTxO <> walletUTxO) partialTx
+    estimateScriptsCost pparams systemStart epochInfo utxo partialTx
```

Comment:
No that big a deal but I don't see value in extracting this `utxo` variable. We do not use it anywhere else (unless I'm missing something) and the name `utxo` does not bring more meaning to the code.
Maybe that could be nice to actually keep this variable but give it a more meaningful name. Since I'm not sure I understand its usage here, I'm not able to make a suggestion, sorry.